### ISSUE TITLE
Web proxy data not tagged correctly

### ISSUE THREAD

**Issue by automine** [PaloAltoNetworks/SplunkforPaloAltoNetworks#37]
Currently the web proxy data is not being tagged correctly. It is being tagged as "web" but not "proxy", so while it does reach the correct data model, it doesn't reach the Proxy node. Here is what I am currently using for this:
`eventtypes.conf`

```
[pan_proxy]
search = sourcetype=pan:threat log_subtype="url"
```

`tags.conf`

```
[eventtype=pan_proxy]
web = enabled
proxy = enabled
```

**Comment by btorresgil** [comment]
Hello, thanks for opening an issue.  Are you using Add-on version 3.7.0, or a different version?

Since the firewall is not technically a proxy, it was recommended to us to only tag as `web`.  But I'm interested to hear how this affects your data.  Can you help me understand what is missing by not having the `proxy` tag?

**Comment by automine** [comment]
Yes, it isn't technically a proxy, however, c'mon, how many of your customers are using it to replace a proxy? :)

It behaves a lot like a proxy, in that it accepts/blocks web traffic, and logs that information. If it wasn't performing the blocking action, I could see making it more like web logging.

In terms of what is missing, really I'm being selfish as I use the Web.Proxy data model here: https://github.com/automine/web_proxies

**Comment by btorresgil** [comment]
Fair enough, we'll do some research and consider this for Add-on 3.7.1.  Will let you know here either way.  Thanks for the input!

**Comment by shadeflayer** [comment]
I concur with Automine.  PA Sales sold the proxy capability as a basic proxy solution, thereby convincing my organization to skip Websense/Forcepoint and use PA instead.  In testing the PA's do provide most of the same basic functionality.  Maybe not all the bells and whistles, but in a hard core, practical environment those are not necessary to protect the environment.  So I am looking forward to significant improvements in this app to provide more robust proxy dashboards.  One thing I really need to see is what a user is doing with their time, and the amount of time by category, and subcategory.  Hint Hint...  

FYI - Check out the App for Web Proxies on Splunkbase for some ideas.  The developer assisted me in getting the app going using the PA logs.  Looks sexy I must say :)

**Comment by btorresgil** [comment]
@shadeflayer Thanks for the comments and the tip on the Web Proxy App.  I will definitely check that out and make sure our logs fit well.

**Comment by shadeflayer** [comment]
@btorresgil - I emailed you yesterday and this morning.  Not sure if you have seen it yet.  Need assistance with the core PA app.  Odd behavior post Splunk 6.5 upgrade.

**Comment by bogeybit** [comment]
We also made the changes in our environment that @automine suggested to work with the Web.Proxy data model. In our case we do use URL filtering as a proxy even though it technically is not a "proxy", the usage and log event format is very similar.

**Comment by Kevin-Manson** [comment]
+1 On this issue, we have been adding the proxy tab to all of our clients.

**Comment by btorresgil** [comment]
Resolved in Add-on 6.0.0
